File: Desafios JS Intermediários/desafio3.js
/*
Entrada
A entrada consiste de múltiplos casos de teste, e cada um consiste em duas linhas: A primeira linha contém um inteiro L (1 ≤ L ≤ 500) representando o número de tartarugas do grupo, e a segunda linha contém L inteiros Vi (1 ≤ Vi ≤ 50) representando as velocidades de cada tartaruga do grupo.

Saída
Para cada caso de teste, imprima uma única linha indicando o nível de velocidade da tartaruga mais veloz do grupo.

*/


//TODO: Complete os espaços em branco com uma possível resolução para o problema.
for(let j = 0; j<3;j++)
{
  let number = parseInt(gets()); // qtd tartarugas
  let velocity = gets().split(" "); // suas velocidades
  let max = 0;
  let level = 0;

  for(i = 0 ; i < number; i++){
    if(max < parseInt(velocity[i])){ //se 0 menor que 10
      max = parseInt(velocity[i]); // maximo vira 10
    }
  }
  
  if(max < 10 ){
    level = 1;
  }
  else if(max >= 10 && max < 20 ){
    level = 2;
  } 
  else{
    level = 3;
  }
  console.log(level);
}

File: Desafio C# Intermediários/desafio3.cs
/*

Entrada
Cada caso de teste inicia com um inteiro N, indicando quantos minutos faltam para o final do expediente (2 <= N <= 100).

Em seguida haverá dois inteiros A e B, indicando quantos minutos são necessários para fabricar os dois presentes que Dobby precisa fabricar (1 <= A, B <= 100).

Saída
Imprima uma linha, contendo a frase "Farei hoje!" caso seja possível fabricar os dois presentes antes do final do expediente, ou "Deixa para amanha!" caso contrário.

*/


using System;

//TODO: Complete os espaços em branco com uma possível solução para o problema. 
class MinhaClasse {
  public static void Main (string[] args) {
    string[] line1 = Console.ReadLine().Split(" "); // recebe a primeira informaçãp
    string[] line2 = Console.ReadLine().Split(" "); // recebe a segunda informação bruta, transformando em um array
    
    
    int finalExpediente = int.Parse(line1[0]);
    

    int presente1 = int.Parse(    line2[0]        );
    int presente2 = int.Parse(      line2[1]      );
    
    
    int totalNecessario =  presente1 + presente2;
    
    if (      totalNecessario > finalExpediente                       )
    {
      Console.WriteLine("Deixa para amanha!");
    }
    else
    {
      Console.WriteLine("Farei hoje!");
    }
  }
}